scripts/test/Muon/grouping_tab/grouping_table_presenter_test.py

```diff
@@ -402,6 +402,36 @@ def test_when_adding_group_to_empty_table_it_is_selected(self):
 
         self.assertEquals(self.model.selected_groups, ['group_1'])
 
+    def test_update_view_from_model_correctly_adds_warnings_for_invalid_periods(self):
+        self.presenter.validate_periods_list = mock.MagicMock(return_value=RowValid.invalid_for_all_runs)
+        self.presenter._view.add_entry_to_table = mock.MagicMock()
+        self.presenter.add_group_to_model(MuonGroup('group_1', [1,2,3,4], [3]))
```

Comment:
can we have some explanation for what the numbers mean?

Reply:
I'll switch to the below to clarify things.
```
self.presenter.add_group_to_model(MuonGroup(name='group_1', detectors=[1,2,3,4], periods=[3]))
```